tests/bundler.js

```diff
@@ -167,6 +179,39 @@ describe( 'bundler', () => {
 	} );
 } );
 
+describe( 'bundler/jsonPackage', () => {
+	before( () => {
+		rimraf( 'tests/fixtures/jsonPackage/dist' );
+	} );
+
+	after( () => {
+		rimraf( 'tests/fixtures/jsonPackage/dist' );
+	} );
+
+	// #155
+	it( 'should load JSON file', () => {
+
+		function testExecution( filename ) {
+			const codeES5 = readFileSync( filename ).toString();
+			const consoleSpy = spy( console, 'log' );
+
+			// I'm so sorry!
+			eval( codeES5 );
+
+			expect( consoleSpy.calledWithExactly( 'Piotr Kowalski' ) ).to.be.true;
+			expect( consoleSpy.callCount ).to.equal( 1 );
```

Comment:
Actually the project uses [`sinon-chai`](https://github.com/domenic/sinon-chai), so it can be written in a more elegant manner:

```suggestion
			expect( consoleSpy ).to.have.been.calledWithExactly( 'Piotr Kowalski' );
			expect( consoleSpy ).to.have.been.calledOnce;
```